app/experimenter/templates/experiments/section_rollout.html

```diff
@@ -11,16 +11,29 @@
   <p>{{ experiment.design|urlize|linebreaks }}</p>
   {% if experiment.is_pref_rollout %}
   <p><strong>Prefs:</strong></p>
-    {% for preference in experiment.preferences.all %}
-      <div class="border-left-grey pl-3 ml-2 mb-3">
-       {% if preference.is_json_string_type %}
-        <p><strong> {{ preference.pref_name }} = <pre class="text-info">{{ preference.pref_value|as_json }}</pre></strong></p>
-       {% else %}
-        <p><strong> {{ preference.pref_name }} = <span class="text-info">{{ preference.pref_value }}</span></strong></p>
-       {% endif %}
-       <p>Pref Type: <span class="text-info">{{ preference.pref_type }}</span></p>
-      </div>
-    {% endfor %}
+  <table class="table table-bordered">
+  <thead>
+    <tr>
+      <th scope="col">Name</th>
+      <th scope="col">Value</th>
+      <th scope="col">Type</th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for preference in experiment.preferences.all %}
+    <tr>
+      <td>{{ preference.pref_name }}</td>
+      {% if preference.is_json_string_type %}
+        <td><pre class="text-info">{{ preference.pref_value|as_json }}</pre></td>
+      {% else %}
+        <td><pre class="text-info">{{ preference.pref_value }}</pre></td>
+      {% endif %}
+
+      <td>{{ preference.pref_type }}</td>
+    </tr>
+  {% endfor %}
+  </tbody>
+  </table>
```

Comment:
Also I think you can just fold these two cases together and get rid of the 'is_multipref' check altogether?

Reply:
so combine multipref and rollout?

Reply:
Sorry I was looking at the wrong thing, I was thinking of folding the single pref and multi pref casese together but you can't, nevermind me!